foundry/cameraPopoutAppWrapper.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Abstraction of the Application interface to be used with the Draggable class as a substitute for the app
+ * This class will represent one popout feed window and handle its positioning and draggability
+ */
+declare class CameraPopoutAppWrapper {
+
+  /**
+   * The CameraViews application that this popout belongs to
+   */
+  view: CameraViews;
+
+  /**
+   * The div element of this specific popout window
+   */
+  element: JQuery;
+
+  /**
+   * ID of the user this popout belongs to
+   */
+  userId: string;
+
+  /**
+   * @param view- The CameraViews application that this popout belongs to
+   * @param userId- ID of the user this popout belongs to
+   * @param element- The div element of this specific popout window
+   */
+  constructor(view: CameraViews, userId: string, element: JQuery);
+
+  /**
+   * Get the current position of this popout window
+   */
+  get position(): Application.Position;
+
+  /**
+   * Sets the position of this popout window
+   */
+  setPosition(position: Application.Position): void;
+
+  /**
+   * The resize event, currently unused
```

Comment:
Please don't add custom TSDoc text. We want to stick as close as possible to foundry for these cases. If you want to add a remark, that's OK, just use
```
  /**
   * @remarks currently unused
   */
 ```
 Also are you actually sure that this really is unused (as opposed to it just not doing anything)?

Reply:
I don't know for sure. I removed my comment